tests/test.py

```diff
@@ -0,0 +1,38 @@
+import time
+import logging
+import os
+import subprocess
+
+from avocado import VERSION
+from avocado import Test
+
+class MinishiftISOTest(Test):
+
+    def setUp(self):
+        ''' Test Setup '''
+        self.test_DIR = os.path.dirname(os.path.realpath(__file__))
+        self.log.info("################################################################")
+        self.log.info("Avocado version : %s" % VERSION)
+        self.log.info("################################################################")
+
+    def test_minishift_centos_iso_build(self):
+        ''' minishift centos iso build test '''
+        self.log.info("Testing minishift centos iso build ...")
+        process = subprocess.Popen("make", shell=True, stdout=subprocess.PIPE)
+        process.wait()
+        self.assertEqual(0, process.returncode)
+
+    def test_minishift_centos_iso_file(self):
+        ''' minishift centos iso file existence test '''
+        self.log.info("Testing minishift centos iso file existence ...")
+        cmd = "test -e %s/../build/minishift-centos.iso" % self.test_DIR
+        process = subprocess.Popen(cmd, shell=True, stdout=subprocess.PIPE)
+        process.wait()
+        self.assertEqual(0, process.returncode)
+
+    def test_build_cleanup(self):
+        ''' minishift iso build clean up test '''
+        self.log.info("Testing minishift build cleanup process ...")
+        process = subprocess.Popen("make clean", shell=True, stdout=subprocess.PIPE)
```

Comment:
@hferentschik Ya delete the ISO which got created during test. I hope that is what tearDown step is suppose to, right?